theme/less/_header.less

```diff
@@ -0,0 +1,63 @@
+#header-top
+{
+  padding: 25px;
+}
+
+header label select
+{
+  width: 163px;
+  color: @secondary;
+  font-size: 24px;
+  border: 0;
+  opacity: 0.7;
```

Comment:
Detail: As mentioned here https://github.com/Privacy-Managers/privacy-manager-website/pull/5#discussion_r475206869 let's not use opacity for adjusting color.

Reply:
Done.